File: corpus/2018q1/_p2q1_ok/_check/B;0089.java

import java.util.Scanner;
public class P2Q1 {

    public static void main(String[] args) {
        Scanner entrada = new Scanner (System.in);

        int V[] = new int [135];
        int i, k = 20;
        System.out.println ("Digite os valores do vetor");
        for ( i = 0; i < 135; i++){
        V[i]= entrada.nextInt();
        }

        comparaVetor (V, i, k);

        boolean returncompara = comparaVetor(V,i,k);
        if (returncompara = true){
            System.out.println ("Existe número maior que 20 nesse vetor.");
        }else{
            System.out.println("Não existe número maior que 20 nesse vetor.");
        }
    }
    public static boolean comparaVetor(int v[], int n, int k){
        boolean x = true;
            for (int i = 0; i < n; i++){
            if (v[i] < k){
                x = false;
            }
        } 
      return x;
    }
}

File: corpus/2018q1/_p2q1_ok/_check/B;0089.py
Scanner entrada = new Scanner (System.in)
int i, k = 20
print ("Digite os valores do vetor")
for 0 in range(0, 135):
comparaVetor (V, i, k)
bool returncompara = comparaVetor(V,i,k)
if returncompara = true:
    print ("Existe número maior que 20 nesse vetor.")
    }else{
    print("Não existe número maior que 20 nesse vetor.")
def comparaVetor(int v[], int n, int k):
    bool x = true
    for 0 in range(0, n):
        if v[i] < k:
            x = false
    return x